db/schema.rb

```diff
@@ -225,6 +225,8 @@
     t.bigint "statement_id"
     t.bigint "statement_adjustment_id"
     t.integer "zendesk_ticket_id"
+    t.string "course_identifier"
```

Comment:
I think its sonly foreign keys usually added to events and these 'extra' bits of data can be passed as `modifications` or `metadata` to the event?

Reply:
removed the columns